packages/diracx-web-components/components/Login/LoginForm.tsx

```diff
@@ -129,109 +126,105 @@ export function LoginForm({
 
   return (
     <React.Fragment>
-      <MUIThemeProvider theme={theme}>
-        <CssBaseline />
-
+      <Box
+        sx={{
+          ml: { xs: "5%", md: "30%" },
+          mr: { xs: "5%", md: "30%" },
+        }}
+      >
         <Box
           sx={{
-            ml: { xs: "5%", md: "30%" },
-            mr: { xs: "5%", md: "30%" },
+            width: "100%",
+            display: "flex",
+            justifyContent: "center",
+            paddingTop: "10%",
+            paddingBottom: "10%",
           }}
         >
-          <Box
+          <img src={logoURL} alt="DIRAC logo" width={150} height={150} />
+        </Box>
+        {singleVO ? (
+          <Typography
+            variant="h3"
+            gutterBottom
+            sx={{ textAlign: "center" }}
+            data-testid="h3-vo-name"
+          >
+            {selectedVO}
+          </Typography>
+        ) : (
+          <Autocomplete
+            options={Object.keys(metadata.virtual_organizations)}
+            getOptionLabel={(option) => option}
+            renderInput={(params) => (
+              <TextField
+                {...params}
+                label="Select Virtual Organization"
+                variant="outlined"
+                data-testid="autocomplete-vo-select"
+              />
+            )}
+            value={selectedVO}
+            onChange={handleVOChange}
             sx={{
-              width: "100%",
-              display: "flex",
-              justifyContent: "center",
-              paddingTop: "10%",
-              paddingBottom: "10%",
+              "& .MuiAutocomplete-root": {
+                // Style changes when an option is selected
+                opacity: selectedVO ? 0.5 : 1,
+              },
             }}
-          >
-            <img src={logoURL} alt="DIRAC logo" width={150} height={150} />
-          </Box>
-          {singleVO ? (
+          />
+        )}
+        {selectedVO && (
+          <Box sx={{ mt: 4 }}>
+            <FormControl fullWidth>
+              <InputLabel>Select a Group</InputLabel>
+              <Select
+                name={selectedVO}
+                value={
+                  selectedGroup ||
+                  metadata.virtual_organizations[selectedVO].default_group
+                }
+                label="Select a Group"
```

Comment:
```suggestion
                label="Select your Virtual Organization"
```

Unless the terminology Virtual Organisation/VO is outdated, it might be better to be consistent so people don't get confused

Reply:
The Virtual Organization is selected before: 

https://github.com/DIRACGrid/diracx-web/blob/00b3ecb468249be295fa31e3c012dfe6e581ac2c/packages/diracx-web-components/components/Login/LoginForm.tsx#L168

Here user can select the group within the Virtual Organization (e.g. `VO: LHCb, groups: [lhcb_user, lhcb_admin]`).

Though I could add a `a` in `Select Virtual Organization` (I would not add `your` since a given physical person can be part of different VOs).

What do you think?